modules/monitoring-exploring-the-visualized-metrics.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * monitoring/cluster-monitoring/examining-cluster-metrics.adoc
+
+[id="exploring-the-visualized-metrics_{context}"]
+= Exploring the visualized metrics
+
+After running the queries, the metrics are displayed on the interactive plot. The X axis of the plot represents time. The Y axis represents the metrics values. Each metric is shown as a colored graph. You can manipulate the plot and explore the metrics.
+
+.Procedure
+
+. Initially, all metrics from all enabled queries are shown on the plot. You can select which metrics are shown.
+* To hide all metrics from a query, click {kebab} for the query and click *Hide all series*.
+* To hide a specific metric, go to the query table and click the colored square near the metric name.
+. To zoom into the plot and change the shown time range, do one of the following:
++
+--
+* Visually select the time range by clicking and dragging on the plot horizontally.
+* Use the menu in the left upper corner to select the time range.
+--
++
+To reset the time range, click *Reset Zoom*.
+. To display outputs of all queries at a specific point in time, hold the mouse cursor on the plot at that point. The query outputs will appear in a pop-up box.
```

Comment:
hold the mouse cursor on the plot at that point
what about changing to "move the mouse cursor over the line"?

Reply:
It's enough to just be on the right point in the X axis (time), you can be anywhere in the Y axis (metric value). It's good if the users know that, that's why I used the wording "on the plot at that point [in time]". You don't need to put cursor directly over the graph ("the line").
Not implemented.